libs/icons/changelogs/2023-08-22-changelog.html

```diff
@@ -0,0 +1,173 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>2023-08-22 - {{version}} Changelog</title>
```

Comment:
Not a blocker, but `{{version}}` outputs as text.

Reply:
Yeah, that's still in the works. Which will happen during nightly builds.